src/lib/ChargeStation/configurations/alpitronic-ccv-ocpp-16.ts

```diff
@@ -0,0 +1,80 @@
+import {
+  EventTypes16 as e16,
+  EventTypes as e,
+} from '../eventHandlers/event-types';
+import sendBootNotification from '../eventHandlers/ocpp-16/send-boot-notification';
+import sendHeartbeat from '../eventHandlers/ocpp-16/send-heartbeat';
+import sendHeartbeatDelayed from '../eventHandlers/ocpp-16/send-heartbeat-delayed';
+import sendStopTransaction from '../eventHandlers/ocpp-16/send-stop-transaction';
+import handleTokenRejection from '../eventHandlers/ocpp-16/handle-token-rejection';
+import sendStartTransaction from '../eventHandlers/ocpp-16/send-start-transaction';
+import sendStatusNotificationPreparing from '../eventHandlers/ocpp-16/send-status-notification-preparing';
+import handleStartCharging from '../eventHandlers/ocpp-16/handle-start-charging';
+import handleTransactionStartedUI from '../eventHandlers/ocpp-16/handle-transaction-started-ui';
+import sendStatusNotificationAvailable from '../eventHandlers/ocpp-16/send-status-notification-available';
+import handleTransactionStoppedUI from '../eventHandlers/ocpp-16/handle-transaction-stopped-ui';
+import sendStatusNotificationCharging from '../eventHandlers/ocpp-16/send-status-notification-charging';
+import handleRemoteStartTransaction from '../eventHandlers/ocpp-16/handle-remote-start-transaction';
+import handleRemoteStopTransaction from '../eventHandlers/ocpp-16/handle-remote-stop-transaction';
+import handleGetConfiguration from '../eventHandlers/ocpp-16/handle-get-configuration';
+import handleChangeConfiguration from '../eventHandlers/ocpp-16/handle-change-configuration';
+import handleBootNotificationCallResultReceived from '../eventHandlers/ocpp-16/handle-boot-notification-call-result-received';
+import handleHeartbeatCallResultReceived from '../eventHandlers/ocpp-16/handle-heartbeat-call-result-received';
+import handleStartTransactionCallResultReceived from '../eventHandlers/ocpp-16/handle-start-transaction-call-result-received';
+import handleStopTransactionCallResultReceived from '../eventHandlers/ocpp-16/handle-stop-transaction-call-result-received';
+import sendChargingLimitReached from '../eventHandlers/ocpp-16/send-charging-limit-reached';
+import sendMeterValues from '../eventHandlers/ocpp-16/send-meter-values';
+import handleReset from '../eventHandlers/ocpp-16/handle-reset';
+import handleSetChargingProfile from '../eventHandlers/ocpp-16/handle-set-charging-profile';
+import sendDataTransferAuthorize from 'lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/send-data-transfer-authorize';
+import sendDataTransferReceipt from 'lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/send-data-transfer-receipt';
+import handleDataTransferCallResultReceived from 'lib/ChargeStation/eventHandlers/ocpp-16/ccv-alpitronic/handle-data-transfer-call-result-received';
+
+// This is the default configuration for OCPP 1.6
+// Each key represents an event, and the value represents an array of handlers that will be called when the event is emitted
+export default {
```

Comment:
Could we use some sort of destruct or something to make sure once we add a handler on ocpp 1.6 we also get in here "for free"?

Reply:
The problem is that this is not always desirable. Some stations don't abide to base OCPP. Alpitronic for instance does not send the `Authorize` message. We could create a bare minimum interface of some sort, but I'm not sure how helpful that's going to be.

Reply:
discussed offline. He will implement some sort of destruct:
Eg:
```js
{
...ocpp16,
[eventAlpitronic.X]: ...
[eventAlpitronic.Y]: ...
}
```